pkg/identityserver/user_registry.go

```diff
@@ -693,12 +693,13 @@ func (is *IdentityServer) deleteUser(ctx context.Context, ids *ttnpb.UserIdentif
 		return nil, err
 	}
 	err := is.store.Transact(ctx, func(ctx context.Context, st store.Store) error {
-		err := st.DeleteUser(ctx, ids)
+		// Delete the the user's sessions to enforce logouts.
```

Comment:
I guess this triggered your `WithSoftDelete` changes, right ?

Reply:
Not really, just ended up changing the order and therefore updating the comments.

The reason for changing the order was to keep it the same logic as the purge, where you try to delete all related things before deleting said user.